h3.go

```diff
@@ -107,44 +160,53 @@ func NewLatLng(lat, lng float64) LatLng {
 }
 
 // LatLngToCell returns the Cell at resolution for a geographic coordinate.
-func LatLngToCell(latLng LatLng, resolution int) Cell {
+func LatLngToCell(latLng LatLng, resolution int) (Cell, error) {
 	var i C.H3Index
 
-	C.latLngToCell(latLng.toCPtr(), C.int(resolution), &i)
+	errc := C.latLngToCell(latLng.toCPtr(), C.int(resolution), &i)
+	if errc != C.E_SUCCESS {
+		return Cell(-1), New(int(errc))
+	}
 
-	return Cell(i)
+	return Cell(i), nil
```

Comment:
You can inline the check into the conditional:

```go
if rc := C.latLngToCell(latLng.toCPtr(), C.int(resolution), &i); rc != C.E_SUCCESS {
  return Cell(-1), errFromCode(int(rc))
}
return Cell(i), nil
```

I've also renamed `errc` to `rc` as it is a return code, not necessarily an error.

---

It's a fun idea to introduce sentinel values `Cell(-1)`, but the error we return is sufficient signal that we can keep the amount of change low by continuing to return empty/undefined values on error.